modules/dqt/jsx/components/searchabledropdown.js

```diff
@@ -0,0 +1,177 @@
+import React, {useState} from 'react';
```

Comment:
Why does this need a different SearchableDropdown than the one in Loris?

Reply:
@driusan can't remember like the button component comment. Will figure it out but maybe styling.

Reply:
@driusan this component was the only one I modified somewhat for the redesign. I'm going to work modifying the original SearchableDropdown component in the jsx directory. Then I think I can remove this file.

Reply:
If this had to be modified you can keep it in this PR for now and modify the original one/remove it in a different PR.